server/src/routers/userRouter.js

```diff
@@ -1,21 +1,29 @@
-const express = require('express')
-const path = require('path')
+const express = require("express");
+const path = require("path");
 const router = new express.Router()
 
-const User = require('../models/users')
+const User = require("../models/users")
 
-router.post('/signup', async (req, res) => {
-    console.log('Creating a new account for a new user')
+router.post("/user/signup", async (req, res) => {
     const user = new User(req.body)
-    console.log(req.body)
     try {
         await user.save()
         res.send(user)
```

Comment:
I had say use `res.json(user)` as that makes it more explicit that you want to send a json response.

Reply:
I was just waiting on to add tokens so that I can really start sending json objects. But now I will turn that into json replies then.

Reply:
So you mean you would convert these into json responses once you incorporate tokens into it?